### PR TITLE
Employee

### PR DESCRIPTION
**Pull request by BobHogvall**
employee entity, controller and rrepository

### CODE REVIEW

**Comment by Svartkettil** [comment]
I think I solved it

**Review by figgge — APPROVED** [review]
Good job!

**Review by BobHogvall — COMMENTED** [review]
If I'm not mistaken, we could probably remove the findAllById method we've added to the repository ourself to the auto-generated one. I think we only changed that because we tried to but getOne and getAll in the same method.

File: src/main/java/com/example/springbootproject/controller/EmployeeController.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            throw new IllegalStateException();
+        return repository.save(employee.get());
+    }
+
+    //R
+    @GetMapping("/{id}")
+    List<Employee> getEmployeeById(@PathVariable Long id){
+        if (repository.findById(id).isEmpty())
+            throw new NoSuchElementException();
+        return repository.findAllById(id);
```
Comment (figgge) [inline src/main/java/com/example/springbootproject/controller/EmployeeController.java:41]:
```suggestion
        return repository.findAllById(Collections.singleton(id));
```

File: src/main/java/com/example/springbootproject/repository/EmployeeRepository.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import com.example.springbootproject.entity.Employee;
+import org.springframework.data.repository.ListCrudRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface EmployeeRepository extends ListCrudRepository<Employee, Long> {
+
+    List<Employee> findAllById(Long longs);
```
Comment (figgge) [inline src/main/java/com/example/springbootproject/repository/EmployeeRepository.java:11]:
```suggestion
```

Comment (figgge) [reply src/main/java/com/example/springbootproject/repository/EmployeeRepository.java:11]: Something like this @BobHogvall?

**Pull request merged**